ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -514,6 +514,7 @@ static const func_info_t func_infos[] = {
 	F1("curl_multi_info_read",                  MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_LONG | MAY_BE_ARRAY_OF_OBJECT),
 	F1("curl_share_init",                       MAY_BE_OBJECT),
 	F1("curl_file_create",                      MAY_BE_OBJECT),
+	F1("curl_string_file_create",               MAY_BE_OBJECT),
```

Comment:
This doesn't seem to match the name of the function (`curl_file_create_from_string`).